.github/workflows/ci_ubuntu_2004.yml

```
@@ -0,0 +1,120 @@
+name: CI on Ubuntu 20.04
+
+on:
+  push:
+    branches:
+      # Push events to branches matching refs/heads/master
+      - 'master'
+  pull_request:
+
+concurrency:
+  group: linux-${{ github.event.pull_request.number || github.ref }}
```

```suggestion
  group: linux-old-${{ github.event.pull_request.number || github.ref }}
```
Or whatever. needs to be different from the other linux workflow

---

Can I pick this name freely? I would call it then `ubuntu2004-${}` and `ubuntu2204-${}`

---

Yes, whatever is in front of `${{ github.event.pull_request.number || github.ref }}` can be chosen freely, just needs to be different for each `yaml` workflow file